src/main/java/duke/DukeDataFileException.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Represents a Duke exception when any error occurs related to the data file.
+ */
+public class DukeDataFileException extends DukeException {
+
+    /**
+     * Returns the string representation of the exception.
+     */
+    @Override
+    public String toString() {
```

Comment:
Do overriden methods need javadoc? Else is there a return for this method that is not mentioned in the javadoc?